src/redux/books/books.js

```diff
@@ -0,0 +1,43 @@
+const BOOK_ADDED = 'BOOK_ADDED';
+const BOOK_REMOVED = 'BOOK_REMOVED';
+
+let lastId = 0;
```

Comment:
- [ ] Please follow the ducks pattern in naming your action types. It should be similar to this 👇 
`const BOOK_ADDED = bookstore/books/BOOK_ADDED`;

Reply:
Thank you for your suggestion. I will change that.